src/ml_tooling/data/base_data.py

```diff
@@ -113,6 +113,10 @@ def _load_training_data(self, *args, **kwargs) -> Tuple[pd.DataFrame, DataType]:
 
         return x, y
 
+    @property
+    def features(self):
+        return [feature_name for feature_name in self.x.columns]
```

Comment:
I think we can just do a `self.x.columns.to_list()`? Don't remember the exact syntax